SubjuGator/drivers/sub8_thrust_and_kill_board/sub8_thrust_and_kill_board/simulation.py

```diff
@@ -154,6 +155,6 @@ def on_data(self, data: bytes, can_id: int) -> None:
             packet = ThrustPacket.from_bytes(data)
         elif data[0] == HeartbeatMessage.IDENTIFIER:
             packet = HeartbeatMessage.from_bytes(data)
-            self._last_heartbeat = rospy.Time.now()
+            self._last_heartbeat = self.get_clock().now()
```

Comment:
Yeah, for this, you're going to need to make this class into a node. Otherwise, it will not be able to get the current time.

Reply:
What if I use the Clock().now() command from rclpy.clock. The current time is stored in a variable and the Clock().now() retrieves the current time without the need for a node.